winml/adapter/WinMLAdapter.cpp

```diff
@@ -318,13 +318,13 @@ class WinMLAdapter : public Microsoft::WRL::RuntimeClass<
       IModelProto** model_proto) override try {
     ZeroCopyInputStreamWrapper wrapper(stream_reference);
 
-    auto model_proto_inner = new onnx::ModelProto();
+    auto model_proto_inner = std::make_unique<onnx::ModelProto>();
```

Comment:
This is here just for the exception safety provided by `unique_ptr`. The next line has `THROW_HR_IF_MSG`, and the throw would leak model_proto_inner. Some tests passed invalid streams on purpose and were leaking the entire ModelProto :(